wgpu-hal/src/metal/command.rs

```diff
@@ -296,6 +296,7 @@ impl crate::CommandEncoder<super::Api> for super::CommandEncoder {
         range: Range<u32>,
         buffer: &super::Buffer,
         offset: wgt::BufferAddress,
+        _: wgt::BufferSize, // Metal doesn't support queries that are bigger than a single element are not supported
```

Comment:
comment needs to be adjusted a bit, it says "not supported" twice

Reply:
Oops